source/agora/flash/Node.d

```diff
@@ -1595,10 +1595,17 @@ public class AgoraFlashNode : FlashNode
 
         foreach (pair; this.getManagedKeys())
         {
+            UTXO utxo = this.network.getEnrollmentUTXO();
+            if(utxo == UTXO.init)
+            {
+                log.trace("Couldn't find the UTXO for this node: {}.", pair.key);
+            }
+
```

Comment:
Urgh, the flash layer really shouldn't piggyback on the registry here... I think this will break the flash layer big time.

Reply:
Ah, Sorry. This is the intermediate code, not the final code. I just wanted to upload my immediate code into the GitHub.